### PR TITLE
proposed: status section for the charter template

### PR DESCRIPTION
**Pull request by koalie**

### CODE REVIEW

**Comment by koalie** [comment]
This is re: @tidoust comment in https://github.com/w3c/charter-media-wg/issues/9#issuecomment-479822087

File: charter-template.html [hunk 1]
```diff
@@ -70,6 +70,11 @@
 
     <main>
       <h1 id="title"><i class="todo">[name]</i> (Working|Interest) Group Charter</h1>
+      
+    <div class="info">
+      <p><i class="todo">[status]</i> <i class="todo">[during advance notice]</i> This is a draft charter proposed for discussion. <i class="todo">[during AC review]</i> This draft charter is under W3C Advisory Committee review.</p>
+      <p class="issue"><b>Note:</b>After AC review, this section is removed.</p>
```
Comment (tidoust) [inline charter-template.html:76]: Do we really need different text during the advance notice and during the AC review? It's easy to forget to update charter bits during transition. Do we need to state explicitly in the draft that it is under AC review? If not, "This is a draft charter proposed for discussion and review" would seem like a good middle-ground.

Comment (koalie) [reply charter-template.html:76]: I don't know and I completely defer to you/project/those in the trenches. I've seen status updates modified as the charters progressed, and reflected this. If the middle-ground appeals to you and @plehegar then it's a good replacement.

Comment (svgeesus) [reply charter-template.html:76]: I agree with @tidoust that there is a high chance this falls out of date. I think it is sufficient to say it is a draft.

Comment (koalie) [reply charter-template.html:76]: I couldn't find how to further edit and I'm not sure I did the right thing, but what I did ended up here: https://github.com/koalie/charter-drafts/pull/2

**Comment by wseltzer** [comment]
Let's make the title say "PROPOSED" until the AC review has been completed. That's harder to forget to update.

**Pull request closed without merging**